datacenter/ucp/1.1/high-availability/backups-and-disaster-recovery.md

```diff
@@ -61,6 +61,8 @@ Enter passphrase: secret
 
 ## Restore command
 
+DTR data must be restored to the same release version where the backups were obtained.
```

Comment:
After rephrasing this, can you move this sentence to line 68? That way we will start by explaining what the `restore` command does, and then tell that you should run the same version.